gears/src/baseapp/params.rs

```diff
@@ -171,13 +171,11 @@ impl Default for EvidenceParams {
 
 impl From<inner::EvidenceParams> for EvidenceParams {
     fn from(params: inner::EvidenceParams) -> EvidenceParams {
-        let duration = params
-            .max_age_duration
-            .map(|dur| dur.seconds * SEC_TO_NANO + i64::from(dur.nanos));
-
         EvidenceParams {
             max_age_num_blocks: params.max_age_num_blocks.to_string(),
-            max_age_duration: duration.map(|val| val.to_string()),
+            max_age_duration: params
+                .max_age_duration
+                .map(|d| i128::from(d.duration_nanoseconds()).to_string()),
```

Comment:
It does look strange. However `d.duration_nanoseconds` returns a wrapped `i128`which can be `Into`d to a `i128`. We could implement display on the return type of `d.duartion_nanoseconds` but I'm reluctant to do this since there's no canonical string representations of durations.

Reply:
What about `fn to_i128_string(&self) -> String`?

Reply:
For now I think it's OK. Maybe we can add this in future.